98-topolvm-partition.yaml

```diff
@@ -0,0 +1,17 @@
+apiVersion: machineconfiguration.openshift.io/v1
+kind: MachineConfig
+metadata:
+  labels:
+    machineconfiguration.openshift.io/role: master
+  name: 98-topolvm-partition
+spec:
+  config:
+    ignition:
+      version: 3.4.0
+    storage:
+      disks:
+        - device: /dev/vda
+          partitions:
+            - label: topolvm
+              sizeMiB: 3072
```

Comment:
can we increase the size of this by shrinking the root partition a bit and adding the freed up space to this partition?